### ISSUE TITLE
Why loss is always equal to 0.69

### ISSUE THREAD

**Issue by springli07** [brijeshiitg/Pytorch-implementation-of-SRNet#14]

**Comment by Chao-Xia** [comment]
i have the same problem！

**Comment by springli07** [comment]
please check your data.



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年4月24日(星期一) 晚上10:32
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [brijeshiitg/Pytorch-implementation-of-SRNet] Why loss is always equal to 0.69 (Issue #14)





 
i have the same problem！

**Comment by Chao-Xia** [comment]
thx for your reply. what's the meaning of "check your data" ? I have done data augmentation. but i also have 0.5 acc and 0.69 lose. what should i do later? 谢谢您的回复，您指的检查数据是什么意思啊？我已经做了数据增强。我还需要做什么吗？@springli07

**Comment by Chao-Xia** [comment]
> please check your data.
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: ***@***.***&gt;; 发送时间: 2023年4月24日(星期一) 晚上10:32 收件人: ***@***.***&gt;; 抄送: ***@***.***&gt;; ***@***.***&gt;; 主题: Re: [brijeshiitg/Pytorch-implementation-of-SRNet] Why loss is always equal to 0.69 (Issue #14) i have the same problem！ — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: ***@***.***&gt;

thx for your reply. what's the meaning of "check your data" ? I have done data augmentation. but i also have 0.5 acc and 0.69 lose. what should i do later? 谢谢您的回复，您指的检查数据是什么意思啊？我已经做了数据增强。我还需要做什么吗？@springli07

**Comment by kingkongs7** [comment]
Yeah, I also have same problem. But when I use tensorflow and same data it's normal

**Comment by comewei** [comment]
> > please check your data.
> > […](#)
> > ------------------ 原始邮件 ------------------ 发件人: _**@**_._**>; 发送时间: 2023年4月24日(星期一) 晚上10:32 收件人: _**@**_.**_>; 抄送: _**@**_._**>; _**@**_.**_>; 主题: Re: [brijeshiitg/Pytorch-implementation-of-SRNet] Why loss is always equal to 0.69 (Issue #14) i have the same problem！ — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: _**@**_.***>
> 
> thx for your reply. what's the meaning of "check your data" ? I have done data augmentation. but i also have 0.5 acc and 0.69 lose. what should i do later? 谢谢您的回复，您指的检查数据是什么意思啊？我已经做了数据增强。我还需要做什么吗？@springli07

hello, do you replicate it,can we communicate to talk it.

**Comment by wjyspace** [comment]
i have the same problem！

**Comment by qigang47** [comment]
> i have the same problem！

我想很大的一个问题是现有的数据集的shape是Shape: (512, 512) 而模型中需要的shape是Shape: (256, 256) 再解决这个问题后我 如果能想起来 会给出答案是不是这个问题

**Comment by wangjiahhhhh** [comment]
> i have the same problem！

你好，你解决了吗

**Comment by springli07** [comment]
已经解决，是数据问题，请检查你的数据。使用PSNR测试一遍数据，再进行训练。


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "brijeshiitg/Pytorch-implementation-of-SRNet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年9月13日(星期三) 晚上10:19
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [brijeshiitg/Pytorch-implementation-of-SRNet] Why loss is always equal to 0.69 (Issue #14)





  
i have the same problem！
  
你好，你解决了吗

**Comment by comewei** [comment]
> 已经解决，是数据问题，请检查你的数据。使用PSNR测试一遍数据，再进行训练。
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: "brijeshiitg/Pytorch-implementation-of-SRNet" ***@***.***&gt;; 发送时间:&nbsp;2023年9月13日(星期三) 晚上10:19 ***@***.***&gt;; ***@***.******@***.***&gt;; 主题:&nbsp;Re: [brijeshiitg/Pytorch-implementation-of-SRNet] Why loss is always equal to 0.69 (Issue #14) i have the same problem！ 你好，你解决了吗 — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you were mentioned.Message ID: ***@***.***&gt;

你好，为什么我使用其他模型跑我的数据没啥问题，但是这个却说数据问题。我之前使用yedrounet跑的

**Comment by springli07** [comment]
可以尝试使用更高嵌入率的算法 然后逐步更换嵌入率低的算法 另外尝试使用一些数据增强的方法。Sent from my iPhoneOn Sep 13, 2023, at 22:23, comewei ***@***.***> wrote:﻿

已经解决，是数据问题，请检查你的数据。使用PSNR测试一遍数据，再进行训练。
…
------------------ 原始邮件 ------------------ 发件人: "brijeshiitg/Pytorch-implementation-of-SRNet" @.>; 发送时间: 2023年9月13日(星期三) 晚上10:19 @.>; @.@.>; 主题: Re: [brijeshiitg/Pytorch-implementation-of-SRNet] Why loss is always equal to 0.69 (Issue #14) i have the same problem！ 你好，你解决了吗 — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you were mentioned.Message ID: @.***>

你好，为什么我使用其他模型跑我的数据没啥问题，但是这个却说数据问题。我之前使用yedrounet跑的

**Comment by qigang47** [comment]
> 可以尝试使用更高嵌入率的算法 然后逐步更换嵌入率低的算法 另外尝试使用一些数据增强的方法。Sent from my iPhoneOn Sep 13, 2023, at 22:23, comewei ***@***.***> wrote:﻿ 已经解决，是数据问题，请检查你的数据。使用PSNR测试一遍数据，再进行训练。 … ------------------ 原始邮件 ------------------ 发件人: "brijeshiitg/Pytorch-implementation-of-SRNet" @.>; 发送时间: 2023年9月13日(星期三) 晚上10:19 @.>; @.@.>; 主题: Re: [brijeshiitg/Pytorch-implementation-of-SRNet] Why loss is always equal to 0.69 (Issue #14) i have the same problem！ 你好，你解决了吗 — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you were mentioned.Message ID: @.***> 你好，为什么我使用其他模型跑我的数据没啥问题，但是这个却说数据问题。我之前使用yedrounet跑的 —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

psnr 多少比较合适

**Comment by snowtors** [comment]
> > i have the same problem！
> 
> 我想很大的一个问题是现有的数据集的shape是Shape: (512, 512) 而模型中需要的shape是Shape: (256, 256) 再解决这个问题后我 如果能想起来 会给出答案是不是这个问题

你好请问512和256会有影响吗

**Comment by italoaa** [comment]
What was the problem in the end? I am getting the same thing

**Comment by italoaa** [comment]
Is it a problem of tensorflow vs pytorch?

**Comment by qigang47** [comment]
> Is it a problem of tensorflow vs pytorch?

No you should train the model from 1.0bpp to 0.4bpp,because the signal is hard to get in low bpp

**Comment by springli07** [comment]
请检查隐写算法数据集 有时会失效发自我的 iPhone在 2025年3月15日，02:34，qigang47 ***@***.***> 写道：﻿
  
Is it a problem of tensorflow vs pytorch?

No you should train the model from 1.0bpp to 0.4bpp,because the signal is hard to get in low bpp—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>



qigang47 left a comment (brijeshiitg/Pytorch-implementation-of-SRNet#14)

Is it a problem of tensorflow vs pytorch?

No you should train the model from 1.0bpp to 0.4bpp,because the signal is hard to get in low bpp

**Comment by karlfroldan** [comment]
I get the exact same problem when training 0.4bpp S-Uniward without modifying any of the original code. I have performed data augmentation and paired training to no avail.

**Comment by lzhamb** [comment]
> > i have the same problem！
> 
> 我想很大的一个问题是现有的数据集的shape是Shape: (512, 512) 而模型中需要的shape是Shape: (256, 256) 再解决这个问题后我 如果能想起来 会给出答案是不是这个问题

前辈拟解决了嘛？是什么原因导致的呢，我训练这个模型准确率一直在0.5，能给一些指导嘛？

**Comment by qigang47** [comment]
> > > i have the same problem！
> > 
> > 
> > 我想很大的一个问题是现有的数据集的shape是Shape: (512, 512) 而模型中需要的shape是Shape: (256, 256) 再解决这个问题后我 如果能想起来 会给出答案是不是这个问题
> 
> 前辈拟解决了嘛？是什么原因导致的呢，我训练这个模型准确率一直在0.5，能给一些指导嘛？

这个需要从1.0bpp 往下梯度训练的 不是代码问题

**Comment by qigang47** [comment]
> > > i have the same problem！
> > 
> > 
> > 我想很大的一个问题是现有的数据集的shape是Shape: (512, 512) 而模型中需要的shape是Shape: (256, 256) 再解决这个问题后我 如果能想起来 会给出答案是不是这个问题
> 
> 前辈拟解决了嘛？是什么原因导致的呢，我训练这个模型准确率一直在0.5，能给一些指导嘛？

这方向垃圾了 不建议做

**Comment by lzhamb** [comment]
> > > > i have the same problem！
> > > 
> > > 
> > > 我想很大的一个问题是现有的数据集的shape是Shape: (512, 512) 而模型中需要的shape是Shape: (256, 256) 再解决这个问题后我 如果能想起来 会给出答案是不是这个问题
> > 
> > 
> > 前辈拟解决了嘛？是什么原因导致的呢，我训练这个模型准确率一直在0.5，能给一些指导嘛？
> 
> 这方向垃圾了 不建议做

没办法呀😭,前辈最后您解决这个问题了吗？

**Comment by qigang47** [comment]
> > > > > i have the same problem！
> > > > 
> > > > 
> > > > 我想很大的一个问题是现有的数据集的shape是Shape: (512, 512) 而模型中需要的shape是Shape: (256, 256) 再解决这个问题后我 如果能想起来 会给出答案是不是这个问题
> > > 
> > > 
> > > 前辈拟解决了嘛？是什么原因导致的呢，我训练这个模型准确率一直在0.5，能给一些指导嘛？
> > 
> > 
> > 这方向垃圾了 不建议做
> 
> 没办法呀😭,前辈最后您解决这个问题了吗？

tmd 我都说了 从1.0bpp 隐写率 往下训练 看不懂吗

**Comment by lzhamb** [comment]
> > > > > > i have the same problem！
> > > > > 
> > > > > 
> > > > > 我想很大的一个问题是现有的数据集的shape是Shape: (512, 512) 而模型中需要的shape是Shape: (256, 256) 再解决这个问题后我 如果能想起来 会给出答案是不是这个问题
> > > > 
> > > > 
> > > > 前辈拟解决了嘛？是什么原因导致的呢，我训练这个模型准确率一直在0.5，能给一些指导嘛？
> > > 
> > > 
> > > 这方向垃圾了 不建议做
> > 
> > 
> > 没办法呀😭,前辈最后您解决这个问题了吗？
> 
> tmd 我都说了 从1.0bpp 隐写率 往下训练 看不懂吗

不好意思，刚刚没看到😂
